cli/command/load_balancer.go

```diff
@@ -10,6 +17,449 @@ func NewLoadBalancerCommand(b *CommandBase) *LoadBalancerCommand {
+			},
+			{
+				Name:      "get",
+				Usage:     "Describe load balancer LOADBALANCER_NAME",
+				Action:    l.read,
+				ArgsUsage: "LOADBALANCER_NAME",
+			},
+			{
+				Name:      "healthcheck",
+				Usage:     "View or update the health check of load balancer LOADBALANCER_NAME. If no options are specified, the health check is displayed.",
+				Action:    l.healthcheck,
+				ArgsUsage: "LOADBALANCER_NAME",
+				Flags: []cli.Flag{
+					cli.StringFlag{
+						Name:  "healthcheck-target",
+						Usage: "Health check target in format 'PROTOCOL:PORT' or 'PROTOCOL:PORT/WITH/PATH'",
+					},
+					cli.IntFlag{
+						Name:  "healthcheck-interval",
+						Usage: "Health check interval in seconds",
+					},
+					cli.IntFlag{
+						Name:  "healthcheck-timeout",
+						Usage: "Health check timeout in seconds",
+					},
+					cli.IntFlag{
+						Name:  "healthy-threshold",
+						Usage: "Number of consecutive successes required to count as healthy",
+					},
+					cli.IntFlag{
+						Name:  "unhealthy-threshold",
+						Usage: "Number of consecutive failures required to count as unhealthy",
+					},
+				},
+			},
+			{
+				Name:      "list",
+				Usage:     "List all load balancers",
+				Action:    l.list,
+				ArgsUsage: " ",
+			},
+		},
+	}
+}
+
+func (l *LoadBalancerCommand) addport(c *cli.Context) error {
+	args, err := extractArgs(c.Args(), "LOADBALANCER_NAME", "PORT")
+	if err != nil {
+		return err
+	}
+
+	port, err := parsePort(args["PORT"], c.String("certificate"))
+	if err != nil {
+		return err
+	}
+
+	loadBalancerID, err := l.resolveSingleEntityIDHelper("load_balancer", args["LOADBALANCER_NAME"])
+	if err != nil {
+		return err
+	}
+
+	loadBalancer, err := l.client.ReadLoadBalancer(loadBalancerID)
+	if err != nil {
+		return err
+	}
+
+	ports := append(loadBalancer.Ports, *port)
+	req := models.UpdateLoadBalancerRequest{
+		LoadBalancerID: loadBalancerID,
+		Ports:          &ports,
+	}
+
+	jobID, err := l.client.UpdateLoadBalancer(req)
+	if err != nil {
+		return err
+	}
+
+	onCompleteFn := func(loadBalancerID string) error {
+		loadBalancer, err := l.client.ReadLoadBalancer(loadBalancerID)
+		if err != nil {
+			return err
+		}
+
+		l.printer.StopSpinner()
+		return l.printer.PrintLoadBalancers(loadBalancer)
+	}
+
+	return l.waitOnJobHelper(c, jobID, "Adding port to load balancer", onCompleteFn)
+}
+
+func (l *LoadBalancerCommand) create(c *cli.Context) error {
+	args, err := extractArgs(c.Args(), "ENVIRONMENT_TARGET", "LOADBALANCER_NAME")
+	if err != nil {
+		return err
+	}
+
+	environmentID, err := l.resolveSingleEntityIDHelper("environment", args["ENVIRONMENT_TARGET"])
+	if err != nil {
+		return err
+	}
+
+	ports := []models.Port{}
+	for _, p := range c.StringSlice("port") {
+		port, err := parsePort(p, c.String("certificate"))
+		if err != nil {
+			return err
+		}
+
+		ports = append(ports, *port)
+	}
+
+	if c.String("healthcheck-target") != "" {
+		if _, err := parseTarget(c.String("healthcheck-target")); err != nil {
```

Comment:
[nit] probably more appropriate to name this `validateTarget` and change the signature to only return an error

Reply:
:white_check_mark: